src/fireEvent.js

```diff
@@ -33,12 +33,17 @@ const isPointerEventEnabled = (
   return isPointerEventEnabled(element.parent, true);
 };
 
+const isTouchEvent = (eventName?: string) => {
+  return eventName === 'press';
```

Comment:
Originally, I structured this as `return ['press'].includes(eventName);` to allow easy additions to this allow list, but opted for a simple comparison for now.